packages/stateless/components/NftCard.tsx

```diff
@@ -262,17 +265,43 @@ export const NftCard = forwardRef<HTMLDivElement, NftCardProps>(
               </div>
             )}
 
-            {/* Floor price */}
-            {floorPrice && (
+            {/* Highest offer */}
+            {highestOffer && (
               <div className="space-y-2">
-                <p className="secondary-text">{t('title.floorPrice')}</p>
+                <p className="secondary-text">{t('title.highestOffer')}</p>
+                <div className="body-text space-y-1 font-mono">
+                  {typeof highestOffer.amount === 'number' &&
+                    !isNaN(highestOffer.amount) &&
+                    highestOffer.offerToken && (
+                      <TokenAmountDisplay
+                        amount={convertMicroDenomToDenomWithDecimals(
+                          highestOffer.amount,
+                          highestOffer.offerToken.decimals
+                        )}
+                        decimals={highestOffer.offerToken.decimals}
+                        iconUrl={highestOffer.offerToken.imageUrl}
+                        symbol={highestOffer.offerToken.symbol}
+                      />
+                    )}
+                  {highestOffer.amountUsd && (
+                    <div
+                      className={`caption-text flex flex-row items-center gap-1${
+                        highestOffer.offerToken?.imageUrl ? ' ml-7' : ''
+                      }`}
```

Comment:
we use `clsx` to clean up this common variable class name issue

```suggestion
                      className={clsx(
                        'caption-text flex flex-row items-center gap-1',
                        !!highestOffer.offerToken?.imageUrl && 'ml-7'
                      )}
```

it basically just ignores any falsey argument and combines string arguments into a single string

Reply:
Cool! Learning a lot from this